website/docs/content/table-guidelines/index.md

```diff
@@ -0,0 +1,219 @@
+
+![Icons being used for status](/assets/content/table-guidelines/icon-status.png)
+!!!
+
+### Service icons
+
+Use service icons within a cell to communicate the source or provider of a service.
+
+![Service icon within a table](/assets/content/table-guidelines/with-service-icon.png)
+
+### Grouping
+
+Use icons to communicate commonalities between values or that a value is part of a larger object or hierarchical structure.
+
+![Icons used for grouping in a table](/assets/content/table-guidelines/icon-grouping.png)
+
+### Product branding
+
+Use icons to communicate that a specific item is a HashiCorp product or resource.
+
+![Icon product branding](/assets/content/table-guidelines/icon-product-branding.png)
+
+!!! Dont
+
+Don’t use an [Icon Tile](/components/icon-tile) in place of an icon within a table cell.
+
+![Icon Tile within a table cell](/assets/content/table-guidelines/icon-tile-product-branding.png)
+!!!
+
+### Leading vs. trailing icons
+
+In general, we recommend using **leading icons** because the text following the icon will remain aligned and thus be easier for the user to scan.
+
+Don’t mix and match different icon positions in the same column.
+
+## Links within cells
+
+By default, we recommend using `secondary` [Inline Links](/components/link/inline) within tables, to avoid overloading the UI with the actionable blue color. The `secondary` variant uses the `foreground-strong` color.
+
+To increase the prominence and further differentiate links from other text content, we recommend using a font-weight of `medium`. In code, you can use the CSS helper classes `hds-typography-body-200` and `hds-font-weight-medium`. In Figma, you can use the text style `Body/200/Link`.
+
+![Link example](/assets/content/table-guidelines/link-example.png)
+
+### Multiple links
+
+If a table contains more than one column of links, consider using a `font-weight` of `medium` for the most important links, usually the title of the row or ID. For less important links, use a `font-weight` of `regular`.
+
+![Multiple links within a table](/assets/content/table-guidelines/multiple-links.png)
+
+### Links in long-form content
+
+If a cell contains long-form or descriptive content, use the link style that is most appropriate for the hierarchy and frequency of links within the content. If there are a minimal number of links, `primary` [Inline Links](/components/link/inline) may be appropriate, but if there are many links `secondary` [Inline Links](/components/link/inline) may work better.
+
+![Links in long-form content](/assets/content/table-guidelines/longform-content-links.png)
+
+## Badge usage
+
+Use [Badges](/components/badge) to communicate status and high-priority metadata within tables.
+
+### Badge type
+
+We recommend using `outlined` [Badges](/components/badge) within tables. This variant provides enough differentiation between the component, the value in the cell, and the background of a table row.
```

Comment:
```suggestion
We recommend using `outlined` [Badges](/components/badge) within tables. This variant provides sufficient differentiation between the component, the value in the cell, and the background of a table row without being too heavy.
```

I would consider expanding upon this a bit to better explain. If they use the solid variant instead, for example, is it too heavy? (I added the last part of the suggestion following this but please adjust if it's not quite right.)